.github/workflows/covector.yml

```diff
@@ -92,16 +92,6 @@ jobs:
         # tests on Windows 2019, until we can figure out a way to fix the problem.
         # NOTE: Using Ubuntu 18.04 to provide glibc compatibility. (#588)
         os: [ubuntu-18.04, ubuntu-latest, macos-latest, windows-2019]
```

Comment:
Do we still need ubuntu-18.04 here?

Reply:
I believe we're using 20.04 on `wallet.rs` bindings now so it's probably safe to do that here too

Reply:
It depends if we want to support older glibc versions. I updated it for now, but if users request it, we might need to roll back this change.